### PR TITLE
post request when a test is positive or symptoms

### LINKED ISSUE #23: Add http POST request whenever the user wants to inform for positive test or symptoms
# REQUIREMENTS
-Dispatch the POST when a positive test is registered or when there are sympotms registered.

### PR DESCRIPTION
**Pull request by Joaquin-Niembro**
# Description
I implemented the behavior for dispatching the request wehnever a test is positive or there are symptoms.

### CODE REVIEW

**Comment by JorgeGarciaRa** [comment]
Jorge Ramirez le está invitando a una reunión de Zoom programada.

Tema: sesión 1
Hora: 3 may. 2021 11:00 p. m. Ciudad de México

Unirse a la reunión Zoom
https://us04web.zoom.us/j/73948753309?pwd=V3MzSmxkSCtjampRMVpjbzNWSHhzZz09

ID de reunión: 739 4875 3309
Código de acceso: 619738

File: AlertaCovid/src/utils/constants/api.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 const loginApi = 'https://hidden-cliffs-21927.herokuapp.com/api/user/sign_in?';
 const logOutApi = 'https://hidden-cliffs-21927.herokuapp.com/api/user/sign_out';
-const GetPlace = 'https://hidden-cliffs-21927.herokuapp.com/api/reports/place_reports';
-
-export {loginApi, logOutApi, GetPlace};
+const GetPlace =
+  'https://hidden-cliffs-21927.herokuapp.com/api/reports/place_reports';
+const reportApi = 'https://hidden-cliffs-21927.herokuapp.com/api/incident';
```
Comment (fernandoamz) [inline AlertaCovid/src/utils/constants/api.js]: The URL API should be a constant and only concat the route that you want to use

File: AlertaCovid/src/utils/constants/api.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 const loginApi = 'https://hidden-cliffs-21927.herokuapp.com/api/user/sign_in?';
 const logOutApi = 'https://hidden-cliffs-21927.herokuapp.com/api/user/sign_out';
-const GetPlace = 'https://hidden-cliffs-21927.herokuapp.com/api/reports/place_reports';
-
-export {loginApi, logOutApi, GetPlace};
+const GetPlace =
+  'https://hidden-cliffs-21927.herokuapp.com/api/reports/place_reports';
```
Comment (fernandoamz) [inline AlertaCovid/src/utils/constants/api.js]:
```suggestion
  const URL_API = 'https://hidden-cliffs-21927.herokuapp.com/api/';
```

File: AlertaCovid/src/utils/constants/api.js (hunk 1 above)
Comment (fernandoamz) [inline AlertaCovid/src/utils/constants/api.js]:
```suggestion
const reportApi = `${URL_API}/incident`;
```

**Review by fernandoamz — CHANGES_REQUESTED** [review]

File: AlertaCovid/src/redux/types.js [hunk 3]
```diff
@@ -6,3 +6,4 @@ export const SYMPTOMS_TRUE = 'SYMPTOMS_TRUE';
 export const TEST_POSITIVE = 'TEST_POSITIVE';
 export const ADD_PLACE = 'ADD_PLACE';
 export const NEW_PLACE = 'NEW_PLACE';
+export const REPORT = 'REPORT'
```
Comment (fernandoamz) [inline AlertaCovid/src/redux/types.js:9]:
```suggestion
export const REPORT = 'REPORT';
```

**Review by fernandoamz — APPROVED** [review]
LGTM!

**Review by HRuvalcabaP — APPROVED** [review]
Good job!

**Review by LeyaimMagma — APPROVED** [review]

**Pull request merged**